_submissions/round-12/1/2015-02-26-wrapping-fortran-func-subs-to-python.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2015-02-26
+round: Round 12
+title: Wrapping Fortran functions/subroutines to Python
+author: Horacio Vargas
```

Comment:
Please add a `permalink` field (see comments on your other file, or examples in other files in this directory).